website_blog_group_old_archives/controllers/website_blog.py

```diff
@@ -70,7 +70,7 @@ def blog(self, blog=None, tag=None, page=1, **opt):
         self._date_begin, self._date_end = opt.get('date_begin'), opt.get('date_end')
         self._tag = tag
         result = super(website_blog, self).blog(
-                blog=blog, tag=tag, page=page, **opt)
+                blog=blog, tag=tag, page=page, **opt.get('opt'))
```

Comment:
I get opt dict as this !
opt = {'date_end': u'2015-12-31', 'date_begin': u'2014-12-31'}
I don't know why I was having that issue back then when I created PR 11 !
but still on https://helptest.therp.nl/blog/our-news-1 , the year archiving is not working. i.e. if you click on 2015 to see blogs of 2015 you will get all the blogs from all the years !